src/components/main/RecruitFloat/HoverText.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  RECRUIT_FLOAT,
+  type RecruitStatus,
+} from "../../../constants/recruit/recruit.ko";
+
+interface HoverTextProps {
+  status: RecruitStatus;
+  days?: number;
+}
+
+const HoverText = ({ status, days }: HoverTextProps) => {
+  if (status === "READY")
+    return (
+      <span>
+        {RECRUIT_FLOAT.ECONO_READY_FOR_RECRUIT} D-{days}
+      </span>
+    );
+  else if (status === "OPEN")
+    return <span>{RECRUIT_FLOAT.ECONO_IS_RECRUITING}</span>;
+  else return <span></span>;
+};
```

Comment:
소소한 리펙토링이지만... 이런 형식은 어떠신가요?? days때문에 안될 수 있을수도 있으나... 제안 해봅니다!!
```suggestion
const HoverText = ({ status, days }: HoverTextProps) => {
  const hoverComponents = {
    "READY": 
      <span>
        {RECRUIT_FLOAT.ECONO_READY_FOR_RECRUIT} D-{days}
      </span>,
    "OPEN":
     <span>{RECRUIT_FLOAT.ECONO_IS_RECRUITING}</span>
   }
   return hoverComponents[status] ?? <span></span>;
};
```

Reply:
오 이렇게 쓴다면 이점이 무엇인가요?

Reply:
1. 앞으로 추가될 state들에 대해서 더욱 쉽게 변경이 가능하다.
2. map을 사용하기 때문에 일정한 성능을 꾀하기 좋다.
이거 말고 몇개 있었는데... 지금 기억나는건 이거네용...

Reply:
솔깃하네요 바로 수정갑니다

Reply:
@loopy-lim 타입 에러 때문에 CLOSED도 빈 값을 넣어줬어요

8edc3a9